resources/rest-service/cloudify/migrations/versions/8e8314b1d848_6_2_to_6_3.py

```diff
@@ -259,3 +261,24 @@ def _add_audit_log_indexes():
 def _drop_audit_log_indexes():
     op.drop_index(op.f('audit_log_creator_name_idx'), table_name='audit_log')
     op.drop_index(op.f('audit_log_execution_id_idx'), table_name='audit_log')
+
+
+# flake8: noqa
```

Comment:
Please don't do a bare noqa, it should be targeted to the line it is intended to target.

Reply:
I've reformatted the string in triple quotes (there was no way to put `# noqa` for the specific line since the triple quotes)